### PR TITLE
SITE-919 | Pass fandommobile query param

### PR DESCRIPTION
**Pull request by gitkuba**

### CODE REVIEW

**Review by rybmat — APPROVED** [review]

File: lib/api/routes.js [hunk 1]
```diff
@@ -65,6 +65,8 @@ module.exports = function routes(parsoidConfig, processLogger, parsoidOptions, p
 		res.locals.oldid = req.params.revision || null;
 		// Fandom change
 		res.locals.mobileve = req.query.mobileve && req.query.mobileve !== 'false' ? '1' : '';
+		// SITE-940 Remove after full fandom desktop rollout
+		res.locals.fandomdesktop = req.query.fandomdesktop && req.query.fandomdesktop !== 'false' ? '1' : '';
```
Comment (mszabo-wikia) [inline lib/api/routes.js:69]: Can we instead enable Parsoid's native gallery handling when FD is available?

Comment (gitkuba) [reply lib/api/routes.js:69]: Would it affect our plans to enhance galleries with a slider mode on FD?

Comment (mszabo-wikia) [reply lib/api/routes.js:69]: We don't change the backend rendering right?

Comment (rybmat) [reply lib/api/routes.js:69]: slider does not exist in the MW world so we need to provide our own implementation @mszabo-wikia

**Pull request merged**